### ISSUE TITLE
Need to update install instructions on cgat.org

### ISSUE THREAD

**Issue by mebbert** [CGATOxford/cgat#321]
Hi,

Thank you for supporting CGAT. Just passing my experience along. I spent a couple of hours trying to get CGAT installed according to the cgat website, using `pip install cgat`, etc. I couldn't get anything to work. I finally tried cloning the repository, and it worked great.

I'm on Mac OS Sierra 10.12.3, 2016 MacBook Pro.

**Comment by Acribbs** [comment]
Thanks for the comment.

What were the specific errors you were encountering?

I had the same issue yesterday with getting the code to install using `pip install cgat` on my MacBook Pro too. However, after installing pysam and Boost by `sudo port install boost` it seemed to work.

**Comment by Acribbs** [comment]
Hi @AndreasHeger, I have been looking at trying to recreate the problem I got yesterday with the installation and I have encountered a problem that I didn't notice before. In a clean environment, When I install the latest version of pysam with `pip install pysam`, pysam v0.10.0 is installed (although the install. However when I try to install cgat with `pip install cgat` the install fails with 
`(test) Adams-MBP-5:pysam adamc$ pip install cgat
Collecting cgat
  Using cached CGAT-0.2.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/8f/mxyhzvj14bqfvfd6rj5szyn00000gn/T/pip-build-6uxqi1/cgat/setup.py", line 32, in <module>
        "the CGAT code collection requires pysam to "
    ImportError: the CGAT code collection requires pysam to be installed before running setup.py (pip install pysam)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/8f/mxyhzvj14bqfvfd6rj5szyn00000gn/T/pip-build-6uxqi1/cgat/`

When I install a previous version of pysam `pip install pysam==0.9.1.4` and then run `pip install cgat` it installs fine. 

Any suggestions for whats going on?

**Comment by mebbert** [comment]
I ran into several issues, but the one I couldn't get past was "Error compiling cython file." I kept assuming it must have been a conflict in the python libraries (e.g., pysam, cython, numpy, etc.) between the standard Mac OS install and my `user` install. The `virtualenv` didn't help. Ultimately, I'm guessing it had to do with versioning. I saw there was an issue with pysam 0.8.4. I have 0.10.0 and didn't want to bother trying different versions. But again, using the latest code from github worked.
